hrui/css/hrui.css

```diff
@@ -84,3 +84,10 @@ body:not(.toolbar-drawer) {
   margin-right: 1%;
 }
 
+#civicrm-menu .crm-logo-sm {
```

Comment:
Where can I find this? I can see the `#civicrm-menu` but not the `.crm-logo-sm` class

Reply:
The markup is added by the Navigation.php file here: https://github.com/civicrm/civihr/pull/2264/commits/3c614e59b6f4441a24f0d524225821d504dc3929#diff-ca7c014b54483fa044ce8589d3f7ce51R570

It's then added as part of the menu, as it can be seen here:

![captura de tela de 2017-11-08 09-38-00](https://user-images.githubusercontent.com/388373/32547246-c013c51e-c468-11e7-8661-47c971ba6ebc.png)

Reply:
My bad, hadn't checked out the right branch in CiviHR. It's there now. I tried removing the `!important` and it still seems to work. Is it required? It might be an option to make the selector more specific either. I don't know much about CSS but I have the idea that using `!important` should be the last option.

Reply:
I had some problems when I tried it without the `!important`, but I wouldn't be surprised if I did something wrong. But, to be honest, I  wouldn't care too much about this. In this release, we'll also have the menu updates (https://github.com/civicrm/civihr/pull/2262) which will completely change the menu and override this. Given that, I think it's ok to keep it as is.